lightningd/peer_htlcs.c

```diff
@@ -2148,9 +2148,12 @@ void json_format_forwarding_object(struct json_stream *response,
 {
 	json_object_start(response, fieldname);
 
-	json_add_hex(response, "payment_hash",
-				    cur->payment_hash,
-				    sizeof(*cur->payment_hash));
+	if (cur->payment_hash)
+		json_add_hex(response, "payment_hash",
+					    cur->payment_hash,
+					    sizeof(*cur->payment_hash));
+	else
+		json_add_string(response, "payment_hash", "UNKNOWN");
```

Comment:
Would it be better to give this as `"payment_hash" : null`? Or would that make trouble for most JSON libraries in other languages?

Reply:
Neither.  If you don't know it, you should omit the field.

Reply:
ack